packages/react/src/ui-react/account/__tests__/create-account-action.spec.tsx

```diff
@@ -63,15 +63,16 @@ describe("Create account action", () => {
         await act(() => action.waitForInitialization());
         expect(action.isInitialized).toBe(true);
 
-        const accountNameInput: HTMLInputElement = await screen.findByLabelText(
-            "Account name",
+        const accountInput: HTMLElement[] = await screen.findAllByLabelText(
+            "Account",
```

Comment:
This should be "Account name"

Reply:
(That should let you revert the changes to this file)

Reply:
This test fails with ‘Account name’ because the label of the element is actually ‘Account.’ Is the label actually supposed to be ‘Account name’?
<img width="378" alt="create-account" src="https://github.com/Azure/BatchExplorer/assets/42812319/ce5657ef-0e3f-4d03-b10e-6a1260d87c12">

Reply:
Yup, the label should be "Account name" not "Account". This was the case before since the test was passing.

Reply:
Screenshot from the main branch:

<img width="516" alt="Screenshot 2023-06-23 at 12 36 36 PM" src="https://github.com/Azure/BatchExplorer/assets/509299/2ded21ed-654a-4679-b403-83e79a3ed259">